node-manager/src/event.rs

```diff
@@ -271,14 +274,15 @@ impl LdkEventHandler for EventHandler {
                     0,
                 ));
 
-                match self.persister.read_payment_info(*payment_hash, true) {
+                match self.persister.read_payment_info(*payment_hash, false) {
                     Some(mut saved_payment_info) => {
                         saved_payment_info.status = HTLCStatus::Succeeded;
                         saved_payment_info.preimage = Some(payment_preimage.0);
+                        saved_payment_info.fee_paid_msat = *fee_paid_msat;
                         match self.persister.persist_payment_info(
                             *payment_hash,
                             saved_payment_info,
-                            true,
+                            false,
```

Comment:
i can't believe i screwed some of these up, i thought i triple checked them. nice catch.